tests/test_mode_manager_llm.py

```
@@ -0,0 +1,59 @@
+import os
+import tempfile
```

Ruff (0.12.2)

`tempfile` imported but unused

Remove unused import: `tempfile`

(F401)